Source/Core/Core/Debugger/PPCDebugInterface.cpp

```diff
@@ -19,26 +19,22 @@
 
 std::string PPCDebugInterface::Disassemble(unsigned int address)
 {
-	// Memory::ReadUnchecked_U32 seemed to crash on shutdown
+	// PowerPC::HostRead_U32 seemed to crash on shutdown
 	if (PowerPC::GetState() == PowerPC::CPU_POWERDOWN)
 		return "";
 
-	if (Core::GetState() != Core::CORE_UNINITIALIZED)
+	if (Core::GetState() == Core::CORE_PAUSE)
```

Comment:
Is there any particular reason this was done?

Iit broke /debug's ability to view PPC instructions in the Code window unless the emulation is paused. They're still mapped, but everything is and only viewable if you use PPC vs x86.

Tested with the release prior to this (4.0-5445) to see that it worked there, and then latest build of Dolphin.